### ISSUE TITLE
separator is misspelt 'seperator'

### ISSUE THREAD

**Issue by angrytongan** [vrimar/construct-ui#23]
"separator" is misspelt "sep<ins>**e**</ins>rator" throughout construct-ui. 1.0.0 branch.

Found this after spending an hour trying to figure out why my `Breadcrumb`s weren't working 😂  Fix would be a breaking change; this issue here as a reminder. All good.

```bash
~/repos/construct-ui (1.0|✔)
% git grep seperator
src/_shared/classes.ts:  BREADCRUMB_SEPERATOR: 'cui-breadcrumb-seperator',
src/components/breadcrumb/Breadcrumb.ts:  seperator?: m.Child;
src/components/breadcrumb/Breadcrumb.ts:    const { class: className, seperator = '/', size, ...htmlAttrs } = attrs;
src/components/breadcrumb/Breadcrumb.ts:  private renderChildren(children: m.ChildArray, { seperator }: IBreadcrumbAttrs) {
src/components/breadcrumb/Breadcrumb.ts:        m(`span.${Classes.BREADCRUMB_SEPERATOR}`, seperator)
src/components/breadcrumb/examples/index.ts:        seperator: m(Icon, { name: Icons.CHEVRON_RIGHT })
src/components/breadcrumb/index.scss:  .cui-breadcrumb-seperator {
src/components/breadcrumb/index.scss:  .cui-breadcrumb-seperator {
src/components/breadcrumb/index.spec.ts:    const seperatorLength = el().querySelectorAll(`.${Classes.BREADCRUMB_SEPERATOR}`).length;
src/components/breadcrumb/index.spec.ts:    assert.equal(seperatorLength, 2);
src/components/breadcrumb/index.spec.ts:  it('Renders custom seperator', () => {
src/components/breadcrumb/index.spec.ts:      seperator: m(Icon, { name: Icons.ACTIVITY })
```

**Comment by vrimar** [comment]
Haha good catch, I'll fix this in the next 1.0 branch commit.

**Comment by vrimar** [comment]
Fixed in latest 1.0 commit.
